common/buildcraft/energy/generation/GenLayerAddOilDesert.java

```diff
@@ -34,7 +36,8 @@ public GenLayerAddOilDesert(long worldSeed, long seed, GenLayer parent) {
     protected boolean canReplaceBiome(int biomeId) {
         Biome biome = Biome.getBiomeForId(biomeId);
         if (biome == null) {
-            BCLog.logger.warn("Couldn't find a biome with an ID of " + biomeId);
+            if (BCLib.DEV) //don't spam outside of dev, it's way to noisy
+                BCLog.logger.warn("Couldn't find a biome with an ID of " + biomeId);
```

Comment:
Just removing the log would be fine actually, we don't have any use for it in dev